misc/graylog.conf

```diff
@@ -637,8 +637,8 @@ mongodb_threads_allowed_to_block_multiplier = 5
 # Connection timeout for a configured LDAP server (e. g. ActiveDirectory) in milliseconds.
 #ldap_connection_timeout = 2000
 
-# Disable the use of SIGAR for collecting system stats
-#disable_sigar = false
+# Disable the use of OSHI for collecting system stats
+#disable_oshi = false
```

Comment:
I think we should add this change to the UPGRADING.rst file.

Reply:
And what do you think about using something like `disable_native_system_stats_collector` instead? That way we don't have to rename it again in the future. :wink: